cmd/serv.go

```diff
@@ -144,11 +144,15 @@ func runServ(c *cli.Context) error {
 		}()
 	}
 
-	isWiki := false
-	unitType := models.UnitTypeCode
+	var (
+		isWiki   bool
+		unitType = models.UnitTypeCode
+		unitName = "code"
+	)
 	if strings.HasSuffix(reponame, ".wiki") {
 		isWiki = true
 		unitType = models.UnitTypeWiki
```

Comment:
@lafriks, no,  this line give the unitType as UnitTypeWiki.

Reply:
Oh, you are right